lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -137,9 +140,13 @@
                 }
             ];
 
-            if (_.isArray(authData.providers)) {
-                var accountsSectionData = {
+            var accountsSectionData = [
+                {
                     title: gettext('Connected Accounts'),
+                    subtitle: gettext(
+                        'You can choose to connect social accounts to edX in order to more easily login' +
+                        'when you return to either edX.org or the edX mobile application.'
```

Comment:
You can link your social media accounts to your edX account to make signing in to edx.org and the edX mobile apps easier.